lighthouse-core/audits/manifest-display.js

```diff
@@ -53,9 +53,14 @@ class ManifestDisplay extends Audit {
     return ManifestDisplay.generateAuditResult({
       rawValue: hasRecommendedValue,
       displayValue,
-      debugString: 'Manifest display property should be set.'
+      debugString: this.createDebugString(hasRecommendedValue)
```

Comment:
@ebidel @brendankenny  I was following the pattern I found [here](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/aria-allowed-attr.js#L54), and saw that it was a fairly consistent pattern through the audits.

A couple of ex:
[aria-allowed-attr
](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/aria-allowed-attr.js#L54)
[color-contrast
](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/color-contrast.js#L54)